CONTRIBUTING.md

```diff
@@ -472,28 +520,32 @@ For information about the syntax, see the [mermaid.js documentation](https://mer
 ```md
 {{< diagram >}}
 flowchart TB
-  This --> That
-  That --> There
+This --> That
+That --> There
 {{< /diagram >}}
 ```
 
 ### File system diagrams
+
 Use the `{{< filesystem-diagram >}}` shortcode to create a styled file system
 diagram using a Markdown unordered list.
 
 ##### Example filestsytem diagram shortcode
+
 ```md
 {{< filesystem-diagram >}}
+
 - Dir1/
 - Dir2/
   - ChildDir/
     - Child
   - Child
 - Dir3/
-{{< /filesystem-diagram >}}
+  {{< /filesystem-diagram >}}
```

Comment:
I don't know that this indentation will break the shortcode, but it isn't needed.